doc/intro.md

```diff
@@ -13,25 +13,26 @@ The different methodologies depend on your preferences and/or the data available
 Note: The small area projections take as an input population projections.
 
 
-**The content of the library follows this structure:**
+## Glossary
+Let's define a few terms and acronyms we use throughout the library:
+
+* CCM: Cohort Component Model/Method
+
+* jumpoff year = base year: the first year of projections
```

Comment:
the base year is the last year of historical data used, or jumpoff year - 1
the jumpoff year is the first year of the projection, or base year + 1